frontend-public/src/components/landingPage/LandingPage.js

```diff
@@ -3,12 +3,8 @@ import { Divider, Button, Icon } from "antd";
 
 import QuestionSidebar from "@/components/common/QuestionsSidebar";
 
-var blockquoteStyle = {
-  margin: "1em 0",
-  paddingLeft: ".8em",
-  color: "#6B6363",
-  borderLeft: "4px solid #A09595",
-};
+import { USER_GUIDE, TAILINGS_DOCUMENT } from "@/constants/assets";
+// import tailingsFile from "./assets/downloads/Register of Tailings Storage Facilities and Dams.xlsm";
```

Comment:
Comment to remove?

Everything else looks pretty reasonable to me.

Reply:
Good eye, I'll remove that. Thanks @NWCalvank